drizzlepac/haputils/hla_flag_filter.py

```diff
@@ -354,11 +353,11 @@ def hla_saturation_flags(drizzled_image, flt_list, catalog_name, catalog_data, p
         if channel.lower() in ['sbc', 'hrc']:
             image_ext_list = ["[sci,1]"]
         dq_sat_bit = 256
-    if channel.lower() == 'wfpc2':
```

Comment:
Can you please add a comment explaining a bit the difference between instrument, detector, and channel as they are used in this code?  The wfpc2, in particular, is sometimes referenced as a channel and other times as an instrument.

Reply:
Some comments added in 4183f5e.